go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/spf13/cobra v1.8.1
 	github.com/ssvlabs/eth2-key-manager v1.5.2
 	github.com/ssvlabs/ssv-spec v1.1.3
-	github.com/ssvlabs/ssv/ssvsigner v0.0.0-20250623120703-7787f963c4ad
+	github.com/ssvlabs/ssv/ssvsigner v0.0.0-20250623145104-7b8f742d1eae
```

Comment:
Since it might be the last audit PR we'll merge, when it's ready for merging, can you please make sure both `go.mod`'s reference the latest PR version and include all changes from `main`? 

Let's keep this comment unresolved until we merge the PR to make sure we won't merge before checking this

Reply:
updated, resolving